hpOneView/resources/uncategorized/unmanaged_devices.py

```diff
@@ -0,0 +1,173 @@
+                ' Development LP'
+__license__ = 'MIT'
+__status__ = 'Development'
+
+from hpOneView.resources.resource import ResourceClient
+
+
+class UnmanagedDevices(object):
+    URI = '/rest/unmanaged-devices'
+
+    def __init__(self, con):
+        self._connection = con
+        self._client = ResourceClient(con, self.URI)
+
+    def get_all(self, start=0, count=-1, filter='', query='', sort=''):
+        """
+        Gets a set of unmanaged device resources according to the specified parameters. Filters can be used to get a
+        specific set of unmanaged devices. With no filters specified, the API returns a potentially paginated list of
+        all the unmanaged device resources subject to start/count/sort parameters.
+
+        Args:
+            start:
+                The first item to return, using 0-based indexing.
+                If not specified, the default is 0 - start with the first available item.
+            count:
+                The number of resources to return. A count of -1 requests all the items.
+                The actual number of items in the response may differ from the requested
+                count if the sum of start and count exceed the total number of items.
+            filter:
+                A general filter/query string to narrow the list of items returned. The
+                default is no filter - all resources are returned.
+            query:
+                 A general query string to narrow the list of resources returned. The default
+                 is no query - all resources are returned.
+            sort:
+                The sort order of the returned data set. By default, the sort order is based
+                on create time, with the oldest entry first.
+
+        Returns:
+             list: Unmanaged Devices
+        """
+        return self._client.get_all(start, count, filter=filter, sort=sort, query=query)
+
+    def get(self, id_or_uri):
+        """
+        Gets a single Unmanaged Device resource based upon its uri or id.
+
+        Args:
+            id_or_uri:
+                Could be either the power device id or the uri
+
+        Returns:
+            dict: The Unmanaged Device
+        """
+        return self._client.get(id_or_uri)
+
+    def add(self, information, timeout=-1):
+        """
+        Adds an unmanaged device resource based upon the attributes specified. Use this method to create an unmanaged
+        device to represent resources that consume space within a rack, or consume power from a power delivery device
+        but cannot otherwise be represented by the management appliance.
+
+        Args:
+            information:
+                Unmanaged Device information
+            timeout:
+                Timeout in seconds. Wait task completion by default. The timeout does not abort the operation
+                in OneView, just stops waiting for its completion.
+
+        Returns:
+            dict: added Unmanaged Devices.
```

Comment:
I think you can add just one Unmanaged Device by time
`dict: added Unmanaged Device.`  instead of  `dict: added Unmanaged Devices.`